tools/myst/host/exec.c

```diff
@@ -403,8 +406,75 @@ int exec_action(int argc, const char* argv[], const char* envp[])
 
     unlink(archive_path);
 
+    oe_sgx_enclave_setting_config_data config_data_setting_optional = {
+        {0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15,
+         0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15,
+         0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15,
+         0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15},
+        2,
+        true /* ignore_if_unsupported */};
+    {
+        void* buffer;
+        size_t buffer_size;
+        if (myst_load_file(roothashes[0], (void**)&buffer, &buffer_size) == 0)
```

Comment:
Can this be
`ECHECK(myst_load_file(roothashes[0], (void**)&buffer, &buffer_size));`?

Is `roothashes[0]` optional?